### DISCUSSION TITLE
如何获取工具链的路径

### DISCUSSION THREAD

**Discussion by KellyGodLv** [xmake-io/xmake#4355] [Q&A] [answered]
MDK armcc工具链执行完成后 想在after_build 调用os.exec("fromelf.exe --bin -o "..gen_fi..".fi".." "..out)   
fromelf.exe 在C:\Keil_v5\ARM\ARMCC\bin 
xmake 有办法获取工具链路径 或者 armcc的路径吗？

**Comment by KellyGodLv** [comment]
目前是使用这个方式 不知道有没有更优雅的方式

 import("detect.sdks.find_mdk")
 local mdk = find_mdk()

**Reply by waruqi** [reply]
没

**Reply by waruqi** [reply, marked as answer]
编译器 可以走 target:tool("cc") 取
